### PR TITLE
Added address reuse to `Glossary` > `Address` page

### PR DESCRIPTION
**Pull request by Bosch-0**
Minor addition.

Closes https://github.com/BitcoinDesign/Guide/issues/598

### CODE REVIEW

**Comment by GBKS** [comment]
How about linking this to [this part of the Transaction privacy page](https://bitcoin.design/guide/payments/privacy/#generate-a-new-address-for-each-payment)?

**Comment by Bosch-0** [comment]
I was going to go over and add relevant links to this glossary addition once this was merged. How about adding links to the Transaction privacy page when we add Lightning to it?

**Comment by GBKS** [comment]
So you want to do a second PR on this snippet of text just to add links?

File: guide/glossary/address.md [hunk 1]
```diff
@@ -99,6 +99,10 @@ The receiver should then have the ability to switch to a Script or Taproot addre
 
 </div>
 
+## Address reuse
+
+Address reuse is the practice of receiving more than one transaction to a single Bitcoin address. Although there are obvious usability benefits of reusing an address this practice is advised against due to the privacy and security implications of doing so. Not only does it negatively impact the privacy and security of the receiver, it also affects others associated with the address but not necessarily the transactions. Reusing an address makes it easier to correlate transactions with a particular user thus hurting the pseudonymity of the user and thus the overall censorship resistance of the network. More on address re-use [here](https://en.bitcoin.it/wiki/Address_reuse)
```
Comment (pavlenex) [inline guide/glossary/address.md:104]:
```suggestion
Address reuse is the practice of receiving more than one transaction to a single Bitcoin address. Although there are obvious usability benefits of reusing an address, this practice is not recommended due to the negative impact on privacy and security. Reusing an address makes it easier to correlate transactions with a particular user, thus hurting the pseudonymity of the user and thus the overall censorship resistance of the network. More on address re-use [here](https://en.bitcoin.it/wiki/Address_reuse)
```

**Review by GBKS — DISMISSED** [review]
LGTM! Love it!

**Review by GBKS — APPROVED** [review]

**Pull request merged**